### PR TITLE
Convert CrossSigningPanel to TS

### PR DESCRIPTION
**Pull request by dbkr**
Type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://6116ab009e92c92131e05347--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/settings/CrossSigningPanel.tsx [hunk 1]
```diff
@@ -25,64 +25,66 @@ import InteractiveAuthDialog from '../dialogs/InteractiveAuthDialog';
...
+        this.unmounted = true;
         const cli = MatrixClientPeg.get();
         if (!cli) return;
         cli.removeListener("accountData", this.onAccountData);
         cli.removeListener("userTrustStatusChanged", this.onStatusChanged);
         cli.removeListener("crossSigning.keysChanged", this.onStatusChanged);
     }
 
-    onAccountData = (event) => {
+    private onAccountData = (event): void => {
```
Comment (t3chguy) [inline src/components/views/settings/CrossSigningPanel.tsx:66]:
```suggestion
    private onAccountData = (event: MatrixEvent): void => {
```

Comment (t3chguy) [reply src/components/views/settings/CrossSigningPanel.tsx:66]: (and relevant import)

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/views/settings/CrossSigningPanel.tsx (hunk 1 above)
Comment (dbkr) [reply src/components/views/settings/CrossSigningPanel.tsx]: I always miss one :/

**Pull request merged**